static_src/util/cf_api.js

```diff
@@ -198,8 +210,15 @@ export default {
     return this.fetchOne(`/spaces/${spaceGuid}/summary`);
   },
 
-  fetchSpaceEvents(spaceGuid) {
-    return this.fetchAllPages(`/spaces/${spaceGuid}/events`, results => results);
+  fetchSpaceEvents(spaceGuid, options) {
```

Comment:
If we're only passing `appGuid` here right now, why should it be an `objects` object rather then just an optional `appGuid`?